source/en/sdr.rst

```diff
@@ -507,9 +507,9 @@ Interpreting Results
 
     * **d_up.tif**: upslope factor of the index of connectivity (Eq. :eq:`d_up`)
 
-    * **e_prime.tif**: sediment downslope deposition, the amount of sediment from a given pixel that does not reach a stream (Eq. :eq:`eprime`)
+    * **e_prime.tif**: sediment downslope deposition, the amount of sediment from a given pixel that does not reach a stream (Eq. :eq:`eprime`). Units: metric tons/hectare/year.

```

Comment:
```suggestion
    * **e_prime.tif**: sediment downslope deposition, the amount of sediment from a given pixel that does not reach a stream (Eq. :eq:`eprime`). Units: t/(ha*year).

```

Reply:
My usual inclination is to spell things out as much as possible, and consistently. So personally I'd be more in favor of "metric tons/(hectare*year)" (or "metric tons/hectare/year", which reads a little more clearly to me, but let's pick one way of writing it and do it consistently throughout the UG). If I was a beginner with the models, I'm not sure I'd know what "t" stands for.

Reply:
I am convinced and concur with Stacie that it's best to spell out to remove any doubt.